src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -3579,6 +3586,25 @@ impl<T: AstInfo> AstDisplay for RefreshOptionValue<T> {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum ClusterScheduleOptionValue {
+    Manual,
+    Refresh,
+}
+
+impl AstDisplay for ClusterScheduleOptionValue {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        match self {
+            ClusterScheduleOptionValue::Manual => {
+                f.write_str("MANUAL");
+            }
+            ClusterScheduleOptionValue::Refresh => {
+                f.write_str("REFRESH");
```

Comment:
Shouldn't this be `ON REFRESH`? I think as-is the create_sql won't roundtrip

Reply:
Oops, sorry, fixed!